core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -319,41 +279,47 @@ wasi_addr_ip_to_bh_ip_addr_buffer(__wasi_addr_ip_t *addr,
     }
 }
 
+bh_clock_id_t
```

Comment:
```suggestion
bool
```
the bh clock id should be returned through the `out` variable, e.g. `*out = BH_CLOCK_ID_MONOTONIC;`